src/libsystemd/sd-network/network-util.c

```diff
@@ -56,6 +56,15 @@ static const char* const link_carrier_state_table[_LINK_CARRIER_STATE_MAX] = {
 
 DEFINE_STRING_TABLE_LOOKUP(link_carrier_state, LinkCarrierState);
 
+static const char* const link_address_family_table[_ADDRESS_FAMILY_MAX] = {
```

Comment:
Maybe, `link_address_family_table` -> `link_required_address_family_table` is better. `link_address_family_table` is slightly ambiguous.